recipe.yaml

```diff
@@ -0,0 +1,41 @@
+---
+- name: This is a recipe for how to cook with ogr
+  hosts: all
+  tasks:
+  - name: Install all packages needed to hack on ogr.
+    dnf:
+      name:
+      - python3-pip
+      - python3-setuptools
+      - git
+      - python3-setuptools_scm
+      - python3-setuptools_scm_git_archive
+      - python3-wheel  # for bdist_wheel
+  - name: Install latest twine for sake of check command
+    pip:
+      name:
+      - twine  # we need newest twine, b/c of the check command
+      - readme_renderer[md]
+      state: latest
+  - name: Install dependencies for CI watcher
```

Comment:
we won't need this task

Reply:
Oh, I missed that. Thanks.

Reply:
But we need most of the packages, so I've changed that only a bit.